### PR TITLE
SODEV-18171 - chore: wiring up auto publishing. adds namespace @buildcom

### PR DESCRIPTION
**Pull request by natedanner**
https://jira.impdir.com/browse/SODEV-18171

**Problem**
We have a fork of quill-render which is being references through a git branch in package.json

**Solution**
Add a  basic Jenkinsfile to publish this package to Artifactory

**Outcome**
    add `@buildcom/` namespace to package name in `package.json`
    Add Jenkinsfile to run tests and then auto-publish on `master` branch

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env groovy
+//@formatter:off
+def repositoryName = 'quill-render'
+def gitRepoUrl = "https://github.com/buildcom/${repositoryName}.git"
+
+def commitId = null
+String pullRequestId = null
+
+def mavenCommandOptions = "--fail-at-end --batch-mode --quiet --settings settings.xml -Dmaven.repo.local=.localMavenRepo"
```
Comment (sjungling) [inline Jenkinsfile]: unused?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+def testCompletedSuccessfully = false
```
Comment (sjungling) [inline Jenkinsfile]: unused, can remove

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+		}
+		// grab the commitMessage and determine if this is a release
+		def commitMessage = sh returnStdout: true, script: "git log -1 --pretty=%B | cat"
+		isRelease = commitMessage.contains("chore(release)")
+	}
+	stage('Build and Test')
+	{
+		deleteDir()
+		unstash "${repositoryName}-${BUILD_NUMBER}"
+		nodejs(nodeJSInstallationName: 'NodeJS v10+')
```
Comment (sjungling) [inline Jenkinsfile]: i'd create a var for `NodeJS v10+` at the top of the file and use that throughout to make upgrading easier

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "quill-render",
+  "name": "@buildcom/quill-render",
```
Comment (sjungling) [inline package.json]: kinda wondering if we should just lock all dependencies and add a `package-lock.json` ?

File: Jenkinsfile (hunk 1 above)
Comment (natedanner) [reply Jenkinsfile]: yeah i'll nuke

File: package.json (hunk 4 above)
Comment (natedanner) [reply package.json]: done

File: Jenkinsfile (hunk 3 above)
Comment (natedanner) [reply Jenkinsfile]: done

File: Jenkinsfile (hunk 2 above)
Comment (natedanner) [reply Jenkinsfile]: done

File: .github/commit-cop.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+requireTitle: true
+requireCommits: 'all'
```
Comment (zieka) [inline .github/commit-cop.yml]: if they plan on just doing squashes and not merged

```suggestion
requireCommits: 'some'
```

Comment (natedanner) [reply .github/commit-cop.yml]: @sjungling merge commit this branch please

**Pull request merged**